packages/hooks/src/index.d.ts

```diff
@@ -6,3 +6,4 @@ export { default as useConnectModal } from './modal/useConnectModal';
 export { default as useOpenModal } from './modal/useOpenModal';
 export { default as useMutationFlow } from './useMutationFlow';
 export { default as useStorageState } from './useStorageState';
+export { default as useFloatingBarScrollHandler } from './useFloatingBarScrollHandler';
```

Comment:
I think we should put this hook in the layout package since it is a very specific hook